edx_solutions_api_integration/users/views.py

```diff
@@ -315,7 +316,17 @@ def get_queryset(self):
             org_ids = map(int, org_ids.split(','))
             queryset = queryset.filter(organizations__id__in=org_ids).distinct()
 
-        if match == 'partial':
+        if match == 'extendedpartial':
+            if name:
+                queryset = queryset.filter(
+                    Q(profile__name__icontains=name) | Q(first_name__icontains=name) | Q(last_name__icontains=name) |
+                    Q(email__icontains=name) | Q(organizations__display_name__icontains=name)
```

Comment:
why are we applying filter organization display name with user name? `Q(organizations__display_name__icontains=name)`

Reply:
Actually, the value of user name could be any of these like user name,  email and organization as we only have single search field on course participants page to perform search based on these fields.